### PR TITLE
Update getting start guide

### PR DESCRIPTION
**Pull request by Nidhognit**
## PR Description

TBD

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/dev/developer-getting-started-guide.md [hunk 1]
```diff
@@ -90,6 +104,13 @@ After installing, make sure to have a look at [Post-Installation steps and addit
 
 {% endinfo_block %}
 
+### Readme.md
+ - Update project install description.
+ - Update repository link. 
+ - Remove unused information (like vagrant installation if you not use it)
```
Comment (RodinVasily) [inline docs/scos/dev/developer-getting-started-guide.md:110]: "if you don't use it"

File: docs/scos/dev/developer-getting-started-guide.md [hunk 2]
```diff
@@ -160,3 +197,52 @@ Get to know the parts of the Spryker Development Virtual Machine with which we s
...
+
+## 7. Checkers configuration
+### Code sniffer
+Also recommended to update code sniffer to latest version (you also need to change `composer.json` file)
+```bash
+composer update spryker/code-sniffer slevomat/coding-standard --with-dependencies
+```
+
+There are will be some new checks, but mostly everything is easy to fix.
+On a project level you may decide to use your own rules, or exclude something that enamled in spryker by default.
```
Comment (RodinVasily) [inline docs/scos/dev/developer-getting-started-guide.md:217]: enabled in Spryker

**Review by RodinVasily — DISMISSED** [review]

File: docs/scos/dev/developer-getting-started-guide.md [hunk 3]
```diff
@@ -68,6 +68,20 @@ We recommend starting with a Docker SDK environment. This option includes Docker
 It features a lightweight environment that is closer to a production implementation.
 
 To start developing your Spryker in Docker, see [Installing Spryker with Docker](/docs/scos/dev/setup/installing-spryker-with-docker/installing-spryker-with-docker.html).
+#### Deploy file
+For local env we use deploy.dev.yml file (see docimentation how to work with it [https://github.com/spryker/docker-sdk/tree/master/docs/07-deploy-file](https://github.com/spryker/docker-sdk/tree/master/docs/07-deploy-file) )
```
Comment (fscholzspryker) [inline docs/scos/dev/developer-getting-started-guide.md:72]:
```suggestion
For local env we use deploy.dev.yml file (see documentation how to work with it [https://github.com/spryker/docker-sdk/tree/master/docs/07-deploy-file](https://github.com/spryker/docker-sdk/tree/master/docs/07-deploy-file) )
```

**Review by fscholzspryker — CHANGES_REQUESTED** [review]

File: docs/scos/dev/developer-getting-started-guide.md [hunk 4]
```diff
@@ -130,6 +151,22 @@ To configure and customize your Spryker Commerce OS, do the following:
...
+ - `config/install/*`
+ - `data/import/*`
+ - `deploy.dev.yml`
+ - `config_default.php` 
+ - `src/SprykerConfig/CodeBucketConfig.php`
+
+### Data import clean up
+In folder `data/import` you can find a lot of files related to each store, you need to define wich store do you need, and remove all unused files.
+
+Pay attention that default config described there `DataImportConfig::getDefaultYamlConfigPath()` and also should be changed.
```
Comment (lenadoc) [inline docs/scos/dev/developer-getting-started-guide.md:166]: Where specifically is "there" in this context?

File: docs/scos/dev/developer-getting-started-guide.md [hunk 5]
```diff
@@ -130,6 +151,22 @@ To configure and customize your Spryker Commerce OS, do the following:
...
+
+For allowing some stores, you need to edit `CodeBucketConfig::getCodeBuckets()`.
```
Comment (lenadoc) [inline docs/scos/dev/developer-getting-started-guide.md:168]: Is this method also is some file of the data import folder?

File: docs/scos/dev/developer-getting-started-guide.md [hunk 6]
```diff
@@ -160,3 +197,52 @@ Get to know the parts of the Spryker Development Virtual Machine with which we s
 * What is the Spryker DevVM (Development Virtual Machine) and why do we need it?
 * Main Structure
 * Technology Stack: Linux distribution, PHP, Postgres, MySQL, ES, Redis, Queue, Jenkins-->
+
+## 6. CI configuration
```
Comment (lenadoc) [inline docs/scos/dev/developer-getting-started-guide.md:201]: What are the links below for? We need to provide some introduction to specify why we are referring to those links

File: docs/scos/dev/developer-getting-started-guide.md [hunk 7]
```diff
@@ -160,3 +197,52 @@ Get to know the parts of the Spryker Development Virtual Machine with which we s
...
+
+On a project level you can enable 6 level
+```yaml
+vendor/bin/phpstan analyze -l 6 -c phpstan.neon src/
+```
+
+## 8. PhpStorm configuration
+### Plugins
+ - PYZ
+ - Symfony Support
```
Comment (lenadoc) [inline docs/scos/dev/developer-getting-started-guide.md:238]: please explain what you mean here (lines 236-238) and what specifically should be done at this step.

**Pull request merged**